bundles/org.openhab.ui/web/src/css/app.styl

```diff
@@ -270,3 +270,16 @@ html
 
 // .card-backdrop-out
 //   animation card-backdrop-fade-out 400ms forwards
+
+
+// Disable user select causing distracting text selection upon long press on phones/tablets
+.disable-user-select
+  -webkit-user-select none
+  -moz-user-select none
+  -ms-user-select none
+  user-select none
```

Comment:
Normally the vendor prefixed versions are added for you by Stylus so you shouldn't need to set them explicitely.
See for instance `oh-cell.vue`.

Reply:
Fixed.